dev/TreeView/TreeViewItem.xaml

```diff
@@ -8,7 +8,7 @@
     xmlns:contract7NotPresent="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractNotPresent(Windows.Foundation.UniversalApiContract,7)">
     
     <Style TargetType="local:TreeViewItem"
-           BasedOn="{StaticResource ListViewItemRevealStyle}"
```

Comment:
>ListViewItemRevealStyle [](start = 36, length = 23)

we should test this out and see nothing is broken.

Reply:
will do